Dockerfile

```diff
@@ -1,23 +1,30 @@
-FROM jruby:9.0.3-jdk
+FROM alpine:latest
```

Comment:
Using a `latest` tag runs the risk of non-reproducible builds as time goes on. Can you use the latest released tag, I think it's `3.4`?